spec/javascripts/app/views/flash_messages_view-spec.js

```diff
@@ -0,0 +1,14 @@
+describe("app.views.FlashMessages", function() {
+  var flashMessages = new app.views.FlashMessages();
+
+  describe("render", function() {
+    it("creates a new div for the message and calls flashes.animateMessages", function() {
+      spyOn(flashMessages, "success");
+      flashMessages.success("success!");
+      expect(flashMessages.success).toHaveBeenCalled();
+      spyOn(flashMessages, "error");
+      flashMessages.error("error!");
+      expect(flashMessages.error).toHaveBeenCalled();
```

Comment:
I'd like to see some tests for the three functions you wrote. These tests don't do anything valuable.